__tests__/itinerary.test.js

```diff
@@ -0,0 +1,18 @@
+const Itinerary = require('../src/itinerary.js')
+
+describe('Itinerary class and properties', () => {
+   const port1 = jest.fn(), port2 = jest.fn();
+   let itinerary = new Itinerary([port1, port2]);
+   
+   describe('Itinerary class', () => {
```

Comment:
you don't need the nested describe blocks, if there is only one test case under each one...